defaults/main.yml

```diff
@@ -6,10 +6,14 @@ matterbridge_cont_vol: '/docker/{{ matterbridge_cont_name }}'
 matterbridge_debug: false
 matterbridge_nick: bridge
 matterbridge_ens_name: bridge.stateofus.eth
+matterbridge_replace_messages: []
```

Comment:
I think you just [cargo-culted](https://en.wikipedia.org/wiki/Cargo_cult) that line. Look at `infra-misc`:
https://github.com/status-im/infra-misc/blob/ce298d52d6368ee173ec9a9999d4570c9554d283/ansible/group_vars/proxy.yml#L82-L85
https://github.com/status-im/infra-misc/blob/ce298d52d6368ee173ec9a9999d4570c9554d283/ansible/group_vars/proxy.yml#L103

It's defined there and used there. Only there. If you're not using `ReplaceMessages` because you're not giving any value to `matterbridge_replace_messages` then what's the point of keeping both?

Reply:
you're right, i copied it from the infra-misc role and forgot to remove it